README.md

```diff
@@ -7,21 +7,21 @@
 
 ## Introduction
 
-FEDB is a NewSQL database optimised for realtime inference and decisioning applications. These applications put real-time features extracted from multiple time windows through a pre-trained model to evaluate new data to support decision making. Existing in-memory databases cost hundreds or even thousands of milliseconds so they cannot meet the requirements of inference and decisioning applications. 
+FEDB is a NewSQL database optimised for online inferencing and decision making applications. These applications feed a pre-trained model with real-time features extracted from multiple time series windows for evaluating new data to support decision making. Existing in-memory databases cost hundreds or even thousands of milliseconds so they cannot meet the requirements of online inferencing and decisioning applications.
 
-FEDB use a double-layered skiplist as the core data structure to store all the data in memory and optimize sql compilation to improve the execution performance.
+FEDB uses a double-layer skiplist as the core data structure. With all the data in memory and extreme compilation optimization of SQL, FEDB significantly reduces execution latency.
 
 - __High Performance__
 
    The benchmark shows that FEDB can be one to two orders of magnitude faster than SingleStore and SAP HANA.
 
 - __SQL Compatible__
 
-   FEDB is compatible with most of ANSI SQL syntax. You can implement your aplications with sqlalchemy or JDBC.
+   FEDB is compatible with most of ANSI SQL syntax. You can implement your applications with SQLAlchemy or JDBC.
 
 - __Online-offline Consistency__
 
-   Machine learning applications developed by FEDB can be launched simply and ensure online and offline consistency, which greatly reduces the cost.
+   Machine learning applications powered by FEDB can be launched easily and ensure online and offline consistency, greatly reducing the cost of landing machine learning scenarios.
```

Comment:
What is the meaning of 'landing machine learning scenarios'? The implementation? Using ML into production?